src/main/java/homework2/Ticket.java

```diff
@@ -43,10 +43,12 @@ private String getTime() {
         return dateTime;
     }
 
-    public String getId() {
-        return id;
+    public Sector getSector() {
+        return sector;
     }
```

Comment:
just create new method, this can be left. just getter))

Reply:
I this case you remove getId, and add getSector
Just add getSector without removing getId)

Reply:
You are my reviewer again)) Did you add yourself?

Reply:
just automating gitHub after I left a comment.